.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
-node_modules/
-lib/
+node_modules
+flow-typed
```

Comment:
I'm not sure since I don't use this, but my guess is that since this is 100% auto-generated & `yarn.lock` pins both `flow-typed` + every package, the "libdefs" would be the same every time...? 
(Plus, already these things clog up PR diffs lol)